sdk/scheduler/src/main/java/org/apache/mesos/specification/ContainerSpec.java

```diff
@@ -3,11 +3,16 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 
+import java.util.Optional;
+
 /**
  * Spec for defining a Container.
  */
 @JsonDeserialize(as = DefaultContainerSpec.class)
 public interface ContainerSpec {
     @JsonProperty("image-name")
```

Comment:
Overall looking at other instances of `JsonProperty` it looks like underscores are used more often in spite of a few pre-existing uses of dashes - let's go with underscores?